### DISCUSSION TITLE
next run before last run

### DISCUSSION THREAD

**Discussion by aamster** [apache/airflow#32072] [General]
### Apache Airflow version

2.5.1

### What happened

![Screenshot 2023-06-21 at 7 29 48 PM](https://github.com/apache/airflow/assets/5454341/a2663d60-de8d-4bbb-bd4a-ff8376784c5d)

### How to reproduce

dunno

### Operating System

mac

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
This is at most an interesting discussion to have. And we definitely need more of your time and effort spend on trying to find out what circumstances caused it. Dropping a screenshotw without any context makes it impossible to act on and make any kind of actions. If you really want to help to diagnose and help to investigate and solve possibly a real issue, we need more information.

If you want to treat it as an issue, that somebody will be able to do anything about, more datails need to be provided. Anyone has a a different system, deployment and installation. No-one has a clue what you've done to get there and what your deployment is and how your machines are configured, which time/timezones they have, what datbase you have, which version of the database, what is the time synchronisation you use, how many distributed machines you have or maybe not. 

Without having the information such report is impossible to do anything else with but convert it to a discussion, hoping that the discussion and your time/input will bring it to the level that someone will be able to see it as an issue and possibly diagnose it.

We (as community) look forward to getting more context and information to be able to act on such report.

**Reply by aamster** [reply]
sorry i am unable to spend any more time on investigating this. This is a glaring issue that your team needs to try to reproduce and fix.

**Reply by potiuk** [reply]
Not really. I think you have complete misunderstanding about how open source software is developed.

There is no my team. I have not team. There are 2.500 volunteers that produce software that you can use for free. If you can help those people to diagnose your issue, that's cool. You helped the community to diagnose and solve the issue. 

If you can't then, well it remains a discussion. If there is no-one who can volunteer the time to reproduce and solve it, then it remains a discussion.
